Phonebook.Frontend/src/app/pages/page-information/page-information.component.html

```diff
@@ -8,7 +8,7 @@ <h1 i18n="PageInformationComponent|Title of the page information@@PageInformatio
       Features are still subject to change without further notice and your data may get lost.
     </p>
   </div>
-  <div class="pb-flex pb-center-element">
+  <div class="pb-flex pb-center-element cardcontainer">
```

Comment:
seems like `pb-center-element` is not needed here.

Reply:
why not?
from my point of view it's always applied and also have a important effect

Reply:
I have removed it and it doesn't change anything

Reply:
please remove. Also no use and effect for me

Reply:
it is used to stack the cards vertically for smaller screen sizes

Reply:
that's just relevant for smaller divices

Reply:
it is only centring three divs, that have `flex: 1` so they each take up the whole width... How are they supposed to center?
I just double checked and it does nothing for me...
Please, note that you can't just untick the properties in the developer console like this: 
![image](https://user-images.githubusercontent.com/13590797/67494623-ca7a5000-f679-11e9-9016-ce060f8fcfd9.png)

as this would apply to all elements where `pb-center-element` is used.